types/components/Button.d.ts

```diff
@@ -10,10 +10,16 @@ export interface FabOptions {
 }
 
 export interface TooltipOptions {
+  exitDelay: Number;
+  enterDelay: Number;
+  html: string;
+  margin: Number;
+  inDuration: Number;
+  outDuration: Number;
+  transitionMovement: Number;
```

Comment:
Should these properties be required?

Reply:
`number` in lowercase

Reply:
If you open `types/sample.test.tsx` you can test these types

Reply:
@alextrastero Can you review this once more? Everything should be fine now